_test/types/PageTest.php

```diff
@@ -249,6 +257,28 @@ public function test_ajax_namespace()
         $this->assertEquals([['label' => 'syntax (wiki)', 'value' => 'wiki:syntax']], $page->handleAjax());
     }
 
+    public function test_ajax_namespace_multiple()
+    {
+        global $INPUT;
+
+        $page = new Page(
+            [
+                'autocomplete' => [
+                    'mininput' => 2,
+                    'maxresult' => 5,
+                    'namespace' => '(wiki|foo)',
```

Comment:
Here a regexp is configured without any delimiters. Later on in nsMatch() you differentiate between regexes and non regexes by the use of delimiters?

Reply:
You are right, this is wrong with the current code.

But I followed your suggestion and got rid of the distinction regex / non-regex, so this definition is now correct.